app/backend/wells/management/commands/export.py

```diff
@@ -175,17 +175,18 @@ def generate_files(self, zip_filename, spreadsheet_filename):
  registries_person.person_guid = well.person_responsible_guid
  left join registries_organization on
  registries_organization.org_guid = well.org_of_person_responsible_guid
+ where well.well_publication_status_code = 'Published'
```

Comment:
None of the wells are actually going to have a "Published" status. They're all code to have Null for well_publication_status_code. So really the query has to be limit on "Published" or Null.